example/proxy_conf.yaml

```diff
@@ -27,6 +27,7 @@ CUSTOM_PLUGIN_MODULE_PATHS:
 BACKEND_MODULES:
   - "plugins/backends/saml2_backend.yaml"
   - "plugins/backends/spidsaml2_backend.yaml"
+  - "plugins/backends/ficepsaml2_backend.yaml"
```

Comment:
can we use spidsaml2_backend.yaml only, having configured the global configuration to have both spid and ficep in a single entity?

Reply:
same error, is a refuse from the old approach.